integration-tests/tests/src/tests/sync/sync-as-local.ts

```diff
@@ -33,7 +33,7 @@ describe.skipIf(environment.missingServer, "Synced Realm as local", () => {
   });
 
   before(async function (this: RealmContext) {
-    this.timeout(5000);
+    this.timeout(30 * 1000);
```

Comment:
We have a global timeout of 60000 set by the GHA configuration.  I'm going to remove these timeouts, as they are not long enough